src/mkdocstrings/handlers/base.py

```diff
@@ -158,16 +160,23 @@ def __init__(self, directory: str, theme: str, custom_templates: Optional[str] =
         """
         paths = []
 
-        if custom_templates is not None:
-            paths.append(Path(custom_templates) / directory / theme)
-
         themes_dir = TEMPLATES_DIR / directory
 
         paths.append(themes_dir / theme)
 
-        if self.fallback_theme != "":
+        if self.fallback_theme:
             paths.append(themes_dir / self.fallback_theme)
 
+        for path in paths:
+            css_path = path / "style.css"
+            if css_path.is_file():
+                self.extra_css += "\n" + css_path.read_text(encoding="utf-8")
+                print('!!!', self.extra_css)
```

Comment:
debug line :) ?

Reply:
:thinking: sorry, dropped that now

Was double checking everything :smile: